chef_master/source/install_chef_air_gap.rst

```diff
@@ -148,13 +148,13 @@ Install Chef DK
 
    .. code-block:: bash
 
-      dpkg -i chefdk_2.0.28-1_amd64.deb
+      dpkg -i chefdk_3.2.30-1_amd64.deb
```

Comment:
Will this DK version work with A1?

Reply:
Shouldn't be an issue. Also if we tell them to go to downloads they're going to get a version similar to this. Mostly I just try to update example chef or DK versions when I see them. Docs that mention apps versions from years ago tells me the docs are potentially outdated as well.